apps/src/code-studio/pd/workshop_enrollment/enroll_form.tsx

```diff
@@ -430,23 +491,24 @@ function EnrollForm(props) {
     'Unsure / Still deciding',
   ];
 
-  const csfCourses = Object.keys(CSF_COURSES)
-    .filter(key => key !== 'courses14_accelerated')
```

Comment:
I removed `courses14_accelerated` from the constant `CSF_COURSES` at the top of this file, making it unnecessary to filter here. There are no other references to `CSF_COURSES` that I could find and it's not exported from here. I don't know why this was the original approach though, and could add it back if necessary